packages/astro/src/core/dev/index.ts

```diff
@@ -178,7 +178,6 @@ export class AstroDevServer {
       return [];
     } catch (e) {
       const err = e as Error;
-      viteServer.ssrFixStacktrace(err);
```

Comment:
Optional: this is just a bit of cleanup. We call `ssrFixStacktrace()` after it’s already happened inside `ssr/index.ts`. While this shouldn’t be causing any errors (looking at the code it only seems to add extra missing context to the error), it’s also not needed as far as I can see.